compose/driver/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6.1
+FROM python:3.6.2rc1
```

Comment:
rc2 is being used here because that is what travis uses

remember that this is for development purposes